model_analyzer/result/result_table_manager.py

```diff
@@ -74,7 +75,8 @@ def tabulate_results(self):
 
         # Fill rows in descending order
         for model in self._result_manager.get_model_names():
-            result_heap = self._result_manager.get_model_sorted_results(model)
+            result_heap = deepcopy(
+                self._result_manager.get_model_sorted_results(model))
```

Comment:
Do we need to copy result_heap?

Reply:
Yes. Because the code below pops all the entries off the heap. Otherwise, you have to be sure that this method is the last thing you call, which I don't think is a very "clean" way of coding things.

Reply:
Can we not come up with a better way to iterate through sorted results without popping?